source/site-administrators/studio/rte2-setup.rst

```diff
@@ -192,11 +192,15 @@ We'll load our external plugin (a custom button) and add it to the RTE's toolbar
    .. code-block:: xml
 
       <external_plugins>
-        <my_button><![CDATA[/studio/api/2/plugin/file?siteId=hello&type=tinymce&name=my_button&filename=plugin.js]]></my_button>
+        <my_button><![CDATA[/studio/api/2/plugin/file?siteId={site}&type=tinymce&name=my_button&filename=plugin.js]]></my_button>
       </external_plugins>
 
    |
 
+   where:
+
+      {site}: a macro that inserts the current siteId
+
```

Comment:
Technically, people can set to a specific site too. Would be good to mention you can either use the `{site}` expression or set it to a specific site from where to load the plugin if it's not the current.

Reply:
While technically possible, I don't think that's a good practice. We'll keep it to "best practices."